internal/pkg/gateway/registry/registry.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2021 IBM All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package registry
+
+import (
+	"github.com/hyperledger/fabric/internal/pkg/gateway"
+
+	"github.com/hyperledger/fabric-protos-go/orderer"
+	"github.com/hyperledger/fabric/common/flogging"
+)
+
+type Registry struct {
+	localEndorser gateway.Endorser
+	logger        *flogging.FabricLogger
+}
+
+// Construct a New registry representing the gateway's view of the network
```

Comment:
nit: we have the doc string linters disabled because our code is awful about doc. That said, in general, godoc should be complete sentences ending with a period.